cli/commands.js

```diff
@@ -1,62 +1,66 @@
 module.exports = {
-    app: {
+    'app': {
         default: true,
         desc: 'Create a new JHipster application based on the selected options'
     },
-    aws: {
+    'aws': {
         desc: 'Deploy the current application to Amazon Web Services'
     },
     'ci-cd': {
         desc: 'Create pipeline scripts for popular Continuous Integration/Continuous Deployment tools'
     },
-    client: {
+    'client': {
         desc: 'Create a new JHipster client-side application based on the selected options'
     },
-    cloudfoundry: {
+    'cloudfoundry': {
         desc: 'Generate a `deploy/cloudfoundry` folder with a specific manifest.yml to deploy to Cloud Foundry'
     },
     'docker-compose': {
         desc: 'Create all required Docker deployment configuration for the selected applications'
     },
-    entity: {
+    'entity': {
         argument: ['name'],
         desc: 'Create a new JHipster entity: JPA entity, Spring server-side components and Angular client-side components'
     },
     'export-jdl': {
         argument: ['jdlFile'],
         desc: 'Create a JDL file from the existing entities'
     },
-    heroku: {
+    'heroku': {
         desc: 'Deploy the current application to Heroku'
     },
     'import-jdl': {
         argument: ['jdlFiles...'],
         desc: 'Create entities from the JDL file passed in argument'
     },
-    info: {
+    'info': {
         desc: 'Display information about your current project and system'
     },
-    kubernetes: {
+    'kubernetes': {
         desc: 'Deploy the current application to Kubernetes'
     },
-    languages: {
+    'languages': {
         argument: ['languages...'],
         desc: 'Select languages from a list of available languages. The i18n files will be copied to the /webapp/i18n folder'
     },
-    openshift: {
+    'openshift': {
         desc: 'Deploy the current application to OpenShift'
     },
     'rancher-compose': {
         desc: 'Deploy the current application to Rancher'
     },
-    server: {
+    'server': {
         desc: 'Create a new JHipster server-side application'
     },
-    service: {
+    'spring-service': {
```

Comment:
add `alias : 'service'` here so that old command works as well, else this will be breaking change